include/QBDI/Range.h

```diff
@@ -34,29 +34,42 @@ template<typename T> class Range {
         return a > b ? a : b;
     }
 
+    T _start;  /*!< Range start value. */
```

Comment:
Why moving to private ?

Reply:
`_start` is move to private to keep the consistency of the range. 

``` cpp
Range<rword> t {3, 5};
t._start = 20; // The range {20, 5} isn't valid.
```

With python, I may use `@property` and `@start.setter` but I don't find a c++ equivalent.